### PR TITLE
Add computed property to tell whether database file exists.

### PR DESCRIPTION
**Pull request by nickmshelley**

### CODE REVIEW

**Comment by nickmshelley** [comment]
Tests pass locally.

File: LDSContent/ItemPackage.swift [hunk 1]
```diff
@@ -70,6 +70,15 @@ public class ItemPackage {
         }
     }
     
+    public var databaseExists: Bool {
+        let exists = FileManager.default.fileExists(atPath: url.appendingPathComponent("package.sqlite").path)
+        if !exists {
+            db = nil
```
Comment (nickmshelley) [inline LDSContent/ItemPackage.swift]: Yeah it seemed odd when I put it in, but then I figured that the side effect is that any future calls won't crash, so the benefit seemed to outweigh the oddness. I can take it out if you want though.

Comment (hiltonc) [inline LDSContent/ItemPackage.swift]: I acknowledge that things should not crash. I haven't had time to look into how you could avoid a crash with this, but a side effect of calling a computer property does not seem like a maintainable solution.

Comment (nickmshelley) [inline LDSContent/ItemPackage.swift]: Baleeted!!!

**Comment by stephanheilner** [comment]
Sooooo, is this okay @hiltonc, or do you still have concerns?

**Comment by hiltonc** [comment]
:+1:, but do we still need to fix a crash?

**Comment by nickmshelley** [comment]
The crash should have been fixed in the last commit. This was more of a "just in case, while we're at it" sort of side effect that probably didn't need to be included.

**Pull request merged**